scripts/transpilation/copy-utils.js

```diff
@@ -0,0 +1,54 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const fs = require("fs");
+const { execSync } = require('child_process');
+
+// eslint-disable-next-line no-console
+console.log('Copying bpk-mixins...');
+
+// if(!fs.existsSync('dist/bpk-mixins')) {
+//   execSync(`mkdir dist/bpk-mixins`);
+// }
+
+execSync(`cp -r packages/bpk-mixins dist/bpk-mixins`);
+execSync(`rm -r dist/bpk-mixins/README.md`);
+
+// eslint-disable-next-line no-console
+console.log('bpk-mixins copied.  👍');
+
+
+// eslint-disable-next-line no-console
+console.log('Copying bpk-stylesheets files...');
+
+if(!fs.existsSync('dist/bpk-stylesheets')) {
+  execSync(`mkdir dist/bpk-stylesheets`);
```

Comment:
https://github.com/Skyscanner/backpack/pull/2959/files/6ec072c445f11a53f7c362df4245f24b055d59fb#r1287264390 in that case can we also remove this bit?

Reply:
Good point let me check this one!

Reply:
So the reason it needs to be there and not for the mixins is due to the mixins being inexistance at the time of copy because of the sass-functions.js file gets babeled over.

Whereas the stylesheets one is not getting babel transpiled so the folders never exist so needs to be copied. If I remove that makedir command it just crashes that the folder doesn’t exist so doesn’t complete the process, so why it remians for one and not the other

![Screenshot 2023-08-08 at 16 45 13](https://github.com/Skyscanner/backpack/assets/8831547/f81ecc49-2704-46b2-bf21-e4da6b64f2b4)